tools/myst/host/exec_linux.c

```diff
@@ -276,7 +241,7 @@ static int _enter_kernel(
                 envc,
                 envp,
                 cwd,
-                options->mapping,
+                &options->host_enc_uid_gid_mappings,
```

Comment:
is the structure getting cleaned up before existing function?

Reply:
`options` is a stack variable in the initial `exec_linux_action` function.